### PR TITLE
INTEGRA-3927: created sha-version.txt token generator to track build …

### PR DESCRIPTION
**Pull request by asargsyan999**
Generate sha-version.txt and save project-id, project version and last commit id into this file. This is to allow us to track commits and distinguish between public and private AC/ACL repository commits. 

This can be tested by running `mvn clean package  -DskipTests`

Ticket: 
IS-INTEGRA-3927

### CODE REVIEW

**Review by srobb-anaplan — APPROVED** [review]
APPROVED

**Review by mihaipop03 — APPROVED** [review]
APPROVED

File: java/sha-version.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+ArtifactId: anaplan-connect | Version: 3.0.0 | SHA-CommitId: d5196c86da7a335772243af6338c0b9028f8f46b
```
Comment (skharab-anaplan) [inline java/sha-version.txt:1]: is this the SHA commit of the code? I think we need the SHA for artefact that will be created from mvn deploy or mvn install.

Comment (asargsyan999) [reply java/sha-version.txt:1]: This is the hash of the current commit id, so if you run `git rev-parse HEAD` this id will be returned. I can change the phase to deploy or install to perform this operation upon installation. 
Can you please tell me what you mean by  SHA for artefact?

**Pull request closed without merging**